docs/user/reference/hook-command.md

```diff
@@ -49,188 +49,160 @@ juju exec --unit synapse/0 "relation-set -r 7 --app secret-id=something-else"
 ```
 
 ````
+# Index
+0. [action-fail](#action-fail)
+1. [action-get](#action-get)
+2. [action-log](#action-log)
+3. [action-set](#action-set)
+4. [application-version-set](#application-version-set)
+5. [close-port](#close-port)
+6. [config-get](#config-get)
+7. [credential-get](#credential-get)
+9. [goal-state](#goal-state)
+11. [is-leader](#is-leader)
+12. [juju-log](#juju-log)
+13. [juju-reboot](#juju-reboot)
+14. [leader-get](#leader-get)
+15. [leader-set](#leader-set)
+16. [network-get](#network-get)
+17. [open-port](#open-port)
+18. [opened-ports](#opened-ports)
+19. [relation-get](#relation-get)
+20. [relation-ids](#relation-ids)
+21. [relation-list](#relation-list)
+22. [relation-model-get](#relation-model-get)
+23. [relation-set](#relation-set)
+24. [resource-get](#resource-get)
+25. [secret-add](#secret-add)
+26. [secret-get](#secret-get)
+27. [secret-grant](#secret-grant)
+28. [secret-ids](#secret-ids)
+29. [secret-info-get](#secret-info-get)
+30. [secret-remove](#secret-remove)
+31. [secret-revoke](#secret-revoke)
+32. [secret-set](#secret-set)
+33. [state-delete](#state-delete)
+34. [state-get](#state-get)
+35. [state-set](#state-set)
+36. [status-get](#status-get)
+37. [status-set](#status-set)
+38. [storage-add](#storage-add)
+39. [storage-get](#storage-get)
+40. [storage-list](#storage-list)
+41. [unit-get](#unit-get)
+---
+
+# action-fail
```

Comment:
Here and everywhere else: Hook command sections should start with heading level 2 (##). Any heading level 1 is rendered in the left-hand Navigation, which is not what we want. PS If this is about the `juju documentation` command and the way it's wired to work for CLI commands, where each is printed to a separate doc, we can do separate docs here too -- but they must be nested under something along the lines of "List of hook commands".

![image](https://github.com/user-attachments/assets/573a9c0d-e054-4740-be1e-07490c843073)